nx-X11/programs/Xserver/dix/window.c

```diff
@@ -3059,6 +3066,16 @@ UnmapWindow(register WindowPtr pWin, Bool fromConfigure)
     ScreenPtr pScreen = pWin->drawable.pScreen;
     WindowPtr pLayerWin = pWin;
 
+#ifdef NXAGENT_SERVER
+    #ifdef TEST
+    if (nxagentWindowTopLevel(pWin))
+    {
+      fprintf(stderr, "UnmapWindow: pWin [%p] fromConfigure [%d]\n", pWin,
+                  fromConfigure);
+    }
+    #endif
+#endif
+
```

Comment:
also copyright problem due to license mix in DIX code, GPL-2 code must remain in the nxagent DDX.